sci-physics/geant-vmc/geant-vmc-4.3.6.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils versionator
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI=" http://root.cern.ch/git/geant4_vmc.git"
+else
+	DOWN_PV=$(get_version_component_range 2-)
+	SRC_URI="http://root.cern.ch/download/vmc/geant4_vmc.${DOWN_PV}.tar.gz"
+	SOURCE_PV=$(replace_all_version_separators - ${DOWN_PV})
+	S="${WORKDIR}/geant4_vmc-${SOURCE_PV}"
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="Virtual Monte Carlo Geant4 implementation"
+HOMEPAGE="http://root.cern.ch/root/vmc/VirtualMC.html"
+
+LICENSE="GPL-2"
+SLOT="4"
+IUSE="doc examples geant3 +g4root +mtroot vgm test"
+
+RDEPEND="
+	sci-physics/root:=
+	>=sci-physics/geant-4.10.03:=[opengl,geant3?]
```

Comment:
Please order unconditional deps lexically by package name, this makes it easier to find stuff when there's a lot of it (yeah, I get this is not the case here).